src/DataUtils.cpp

```diff
@@ -0,0 +1,211 @@
+#include "NAS2D/DataUtils.h"
+
+#include "NAS2D/Xml/XmlAttribute.h"
+#include "NAS2D/Xml/XmlElement.h"
+
+#include <algorithm>
+#include <sstream>
+
+#if defined(_WIN32)
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#endif
```

Comment:
With programmatic error reporting (such as exceptions), platform specific code such as this could be removed.